pkg/oauth/handlers/default_auth_handler.go

```diff
@@ -115,11 +115,7 @@ func (authHandler *unionAuthenticationHandler) AuthenticationNeeded(apiClient au
 				w.WriteHeader(http.StatusFound)
 			default:
 				w.WriteHeader(http.StatusUnauthorized)
-				ctx := req.Context()
-				if !ok {
```

Comment:
I'm some 90% sure this should've been `ctx == nil` originally

Reply:
Yes. It does look suspicious. But the functional call `req.Context()` seem to suggest otherwise. How do you think we should proceed on this then?